database/migrations/2025_03_06_110500_retire_workbench.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+
+    private $tables = [
+        'devices_battcat_ora',
+        'devices_dustup_ora',
+        'devices_faults_adjudicated',
+        'devices_faults_batteries_ora_adjudicated',
+        'devices_faults_batteries_ora_opinions',
+        'devices_faults_events',
+        'devices_faults_mobiles_adjudicated',
+        'devices_faults_mobiles_opinions',
+        'devices_faults_mobiles_ora_adjudicated',
+        'devices_faults_mobiles_ora_opinions',
+        'devices_faults_opinions',
+        'devices_faults_printers_ora_adjudicated',
+        'devices_faults_printers_ora_opinions',
+        'devices_faults_tablets_ora_adjudicated',
+        'devices_faults_tablets_ora_opinions',
+        'devices_faults_vacuums_ora_adjudicated',
+        'devices_faults_vacuums_ora_opinions',
+        'devices_misc_adjudicated',
+        'devices_misc_opinions',
+        'devices_mobifix_ora',
+        'devices_printcat_ora',
+        'devices_tabicat_ora',
+        'fault_types',
+        'fault_types_batteries',
+        'fault_types_mobiles',
+        'fault_types_printers',
+        'fault_types_tablets',
+        'fault_types_vacuums',
+        'microtask_demographics',
+        'microtask_surveys',
+    ];
+
+    public function up()
+    {
+        // For each table, rename to 'archived__' + table name.
+        // Using two underscores triggers phpMyAdmin table grouping.
+        foreach ($this->tables as $table) {
+            try {
+                Schema::rename($table, 'archived__' . $table);
```

Comment:
Could we perhaps have them as something like 'z_archived__'?

Just a minor point, as having all of these tables at the beginning of the tables list in phpmyadmin or reporting software will be a slight inconvenience.  That said, I can hide them in e.g. Metabase, so no real problem there.  

I don't mind too much.  What do you think?

Reply:
phpMyAdmin groups them, though, so it's just one line.  I kind of like being obliged to notice we have crud in the DB.  But ask me again and I'll change it.

Reply:
Ah yeah, good point about PMA.  And also agree, having it visible pushes one to remove it.  So, fine as is!